Core/src/org/sleuthkit/autopsy/corecomponents/Installer.java

```diff
@@ -111,4 +114,13 @@ private void setOSXLookAndFeel() {
             UIManager.put(entry.getKey(), entry.getValue());
         });
     }
+    
+    private void setUnixLookAndFeel(){
+        try {
+            //UIManager.setLookAndFeel(UIManager.getCrossPlatformLookAndFeelClassName());
+            UIManager.setLookAndFeel(UIManager.getCrossPlatformLookAndFeelClassName());
+        } catch (ClassNotFoundException | InstantiationException | IllegalAccessException | UnsupportedLookAndFeelException ex) {
+            Exceptions.printStackTrace(ex);
```

Comment:
Please log the error, don't print it to stdout. 

What happens to the LAF if an exception is thrown here?

Reply:
I think we can solve that in the next release.  This is good enough for now.